terraform/README.md

```diff
@@ -1,114 +1,253 @@
+* [Artifact Registry](https://cloud.google.com/artifact-registry)
+  to store GKE Policy Automation tool container image locally
+* [Cloud Asset Inventory](https://cloud.google.com/asset-inventory/docs/overview)
+  to discover GKE clusters in GCP organization, selected folders or projects
+* Optionally, [Cloud Storage](https://cloud.google.com/storage),
+  [Cloud Pub/Sub](https://cloud.google.com/pubsub), or
+  [Security Command Center](https://cloud.google.com/security-command-center)
+  as destinations for cluster evaluation results
 
-Unfortunately, this service is currently only available in one region and not supported by Terraform
-yet. While most of this setup can easily be deployed by running this Terraform script, Cloud Run
-Jobs have to be deployed manually with the gcloud CLI until the Terraform provider is available.
+![GKE Policy Automation infrastructure](../assets/gke-policy-automation-infra.png)
 
-## Before you begin
+---
 
-Before running the script and the additional gcloud commands, please set the following environment
-variables:
+## Prerequisites
 
-```bash
-export TF_VAR_job_name="gke-policy-automation-job"
-```
+* Terraform installed
+* `gcloud` command
+* Exiting project for GKE Policy Automation resources
+* IAM permissions to create resources in the GKE Policy Automation project
+* IAM permissions to create new IAM role bindings on projects, folders or organization levels
+  *(depending on desired cluster discovery or outputs configuration)*
 
-```bash
-export TF_VAR_project_id="YOUR GCP PROJECT ID"
-```
+## Running
 
-```bash
-export TF_VAR_region="YOUR GCP REGION, e.g. europe-west1"
-```
+Provision infrastructure with Terraform:
 
-Adjust the `config.yaml` file according to your preferences. The provided example will use cluster discovery
-on a GCP project, Cloud Storage output for reports and the default set of GKE policies.
+1. Set Terraform configuration variables *(check [examples](#example-configurations)
+   or [inputs](#inputs) below for details)*.
 
-Please check the [User Guide](../docs/user-guide.md) for all available configuration options.
+   Example `tfvars` file:
 
-Please do **NOT** modify the `${PROJECT_ID}` and `${BUCKET_NAME}` placeholders as they will be automatically
-added by Terraform before uploading the file to Secret Manager.
+   ```hcl
+   project_id = "gke-policy-123"
+   region     = "europe-west2"
 
-## What happens behind the scenes
+   discovery {
+     enabled = true
+     projects = ["gke-project-one", "gke-project-two"]
+   }
 
-The Terraform script within this folder enables all required APIs for you and creates necessary
-service accounts and IAM bindings. It also creates the Artifact Registry required by Cloud Run,
-the GCS bucket for storing the reports and a Secret Manager. The Secret Manager is used to provide
-the config.yaml file to the Cloud Run Job, as Cloud Run does not easily support persistent file
-storage.
+   output_cloud_storage = {
```

Comment:
in variables.tf there is `output_storage` - shouldn't it be the same?

Reply:
You got me :-) Thanks, fixing.